deploy/platform/docker/Makefile.in

```diff
@@ -17,4 +17,4 @@
 #
 
 .EXPORT_ALL_VARIABLES:
-FEATURE_FLAGS ?= agent,cluster,so11y,vm-monitor,event,mysql-monitor,nginx-monitor,apisix-monitor,grafana,elasticsearch-monitor,pulsar-monitor
+FEATURE_FLAGS ?= agent,cluster,so11y,vm-monitor,event,mysql-monitor,nginx-monitor,apisix-monitor,grafana,elasticsearch-monitor,pulsar-monitor,rocketmq-monitor,mongodb-monitor,rabbitmq-monitor,postgresql-monitor,satellite,single-node
```

Comment:
This is absolutely wrong as it set both cluster and single-node, which indicate OAP runs in cluster mode and single node mode, which is conflict

Reply:
I think we could set `make deploy.docker FEATURE_FLAGS=cluster,...` or `make deploy.docker FEATURE_FLAGS=single-node,...`

Reply:
I think we should consider to limit default deployment and keep one per kind only.
Otherwise, even currently, most people don't have enough local resources to support this.

Reply:
Keep pulsar or Kafka as MQ default should be good, and MySQL as default DB showcase.

Reply:
It's good, but my idea is that we could all declare it in `FEATURE_FLAGS` and let users choose the required components. I can add the document later https://skywalking.apache.org/docs/skywalking-showcase/next/readme/#docker-compose, how about this way

Reply:
Isn't feature flag provided in the docs already?

Reply:
Yep, the feature flags for different platforms description is already there, could we align it with the way of kubernetes deploy

Reply:
Do you mean align k8s and docker compose deployment? Aren't they aligned? Or do you mean yo align default deployment only?

I am not following.

Reply:
> I think we could set `make deploy.docker FEATURE_FLAGS=cluster,...` or `make deploy.docker FEATURE_FLAGS=single-node,...`

The FEATURE_FLAGS in the Makefile is not a doc, it serves as the default value, so users don't need to set the flags every time and it makes sense for new users to quick start.

Reply:
Right, I got what the two teachers said. I mean align default deployment before. I will fix it later, revert it is ok I think.